genesis/process/metaGenesisBlockCreator.go

```diff
@@ -148,6 +148,11 @@ func createMetaGenesisAfterHardFork(
 	}
 	saveGenesisBodyToStorage(processors.txCoordinator, bodyHandler)
 
+	err = saveGenesisMetaToStorage(arg.Store, arg.Marshalizer, metaHdr)
```

Comment:
Should be critical? saveGenesisBodyToStorage method above just prints some warns inside and returns no error

Reply:
Yes. It is critical. As it is used in compute economics in the first epoch